pkg/packagekit/updates.jsx

```diff
@@ -556,13 +565,14 @@ class OsUpdates extends React.Component {
                                 console.warn("GetTransactionList: failed to read PackageKit transaction roles:", ex.message);
                                 // be robust, try to continue with loading updates anyway
                                 this.initialLoadOrRefresh();
+                                this.loadUpdates();
                             });
                 })
                 .fail(this.handleLoadError);
     }
 
     handleLoadError(ex) {
-        console.error("loading available updates failed:", JSON.stringify(ex));
+        console.warn("loading available updates failed:", JSON.stringify(ex));
```

Comment:
This was actually deliberately an error so that the tests would fail if it updates failed in some case. This is why one test had `allow_browser_errors()`. If you don't like this much, I don't have a strong attachment to it, but it should be a separate commit then.

Reply:
When this is a error, all (or at leat a lot) tests will fail since we now pre-load the updates page.  Hmm.

Reply:
Right, makes sense. Thanks!